### PR TITLE
translate message of self-issued creation

### PR DESCRIPTION
**Pull request by TalebRafiepour**

### CODE REVIEW

File: lib/l10n/arb/app_fr.arb [hunk 1]
```diff
@@ -185,5 +185,6 @@
   "networkErrorUnableToProcess": "Incapacité à traiter la requête",
   "networkErrorUnauthenticated": "Vous devez être authentifié pour accéder à cette ressource",
   "networkErrorUnauthorizedRequest": "Requête non autorisée",
-  "networkErrorUnexpectedError": "Une erreur inattendue s'est produite"
+  "networkErrorUnexpectedError": "Une erreur inattendue s'est produite",
+  "selfIssuedCreatedSuccessfully": "Identifiant auto-émis créé avec succèsy"
```
Comment (hawkbee1) [inline lib/l10n/arb/app_fr.arb]: no "y" at end of this sentence
"Identifiant auto-émis créé avec succès"

**Review by bibash28 — COMMENTED** [review]
BlocProvider.of<WalletCubit>(context)

We have used it in AppWidget.. I think you have to use it in AppWidget rather than using it again in other places...

**Comment by TalebRafiepour** [comment]
> 
@bibash28 
Which line do you mean?

**Comment by bibash28** [comment]
i misread the line... Code is good after I looked the whole file...

**Comment by bibash28** [comment]
Since we already used tha BlocProvider on line 50/...

I think you should use context.read<Self...>() in line 60..

What do you think?

**Review by hawkbee1 — APPROVED** [review]

**Pull request merged**